velox/docs/functions/presto/array.rst

```diff
@@ -284,6 +284,16 @@ Array Functions
 
     Flattens an ``array(array(T))`` to an ``array(T)`` by concatenating the contained arrays.
 
+.. function:: ngrams(array(T)) -> array(array(T))
+
+    Returns n-grams for the array. ::
```

Comment:
perhaps, turn n-grams into a link to a Wikipedia article to allow readers to get context easily: https://en.wikipedia.org/wiki/N-gram